crates/core/Cargo.toml

```
@@ -7,6 +7,7 @@ edition = "2018"
 
 [dependencies]
 aes = "0.6"
+crossbeam = "0.7"
```

I think you only need the channel

```suggestion
crossbeam_channel = "0.7"
```